### PR TITLE
Fixes clipboard sync

### PR DESCRIPTION
**Pull request by ibeckermayer**
See the two commit messages for a detailed description

1. https://github.com/gravitational/webapps/commit/fadc27f9486035bb979d61d2033cfedc09803de1
2. https://github.com/gravitational/webapps/commit/95b3a6e0b4f01b7f8158f4cc273eef94f61a972b

backport to v11/10/9 required

fixes https://github.com/gravitational/teleport/issues/17195

### CODE REVIEW

File: packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx [hunk 1]
```diff
@@ -137,11 +133,28 @@ export default function useTdpClientCanvas(props: Props) {
     e.preventDefault();
     if (handleCapsLock(cli, e)) return;
     cli.sendKeyboardInput(e.code, ButtonState.DOWN);
+
+    // The key codes in the if clause below are those that have been empirically determined not
+    // to count as transient activation events. According to the documentation, a keydown for
+    // the Esc key and any "shortcut key reserved by the user agent"don't count as activation
```
Comment (zmb3) [inline packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx:139]:
```suggestion
    // the Esc key and any "shortcut key reserved by the user agent" don't count as activation
```

**Review by zmb3 — COMMENTED** [review]
No further comments on the code, but I'll give this a test drive later.

File: packages/teleport/src/DesktopSession/useDesktopSession.tsx [hunk 2]
```diff
@@ -56,65 +55,9 @@ export default function useDesktopSession() {
...
-      });
-    } else {
-      setClipboardState({
-        enabled: hasClipboardSharingEnabled,
-        permission: clipboardRWPermission,
-        errorText: '',
-      });
-    }
-  }, [isUsingChrome, hasClipboardSharingEnabled, clipboardRWPermission]);
+  // Becomes user.acl.clipboardSharingEnabled && isUsingChrome
```
Comment (hatched) [inline packages/teleport/src/DesktopSession/useDesktopSession.tsx:59]:
```suggestion
  // Set by result of `user.acl.clipboardSharingEnabled && isUsingChrome` below.
```

**Review by hatched — APPROVED** [review]

**Review by kimlisa — APPROVED** [review]
lgtm, i haven't tested it though, if you'd like help testing can you update with instructions

**Comment by ibeckermayer** [comment]
> lgtm, i haven't tested it though, if you'd like help testing can you update with instructions

@zmb3 has a full setup at his fingertips and is manually double checking, so I think we should be good to go.

**Review by zmb3 — APPROVED** [review]
Looks good, ran tests on Safari and Chrome and saw the expected behavior.

**Pull request merged**